frontend/config/webpack.config.common.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const path = require('path');
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - Parsing error: The keyword 'const' is reserved